content/reference-capabilities/capability-subtyping.md

```diff
@@ -8,39 +8,53 @@ menu:
 toc: true
 ---
 
-Subtyping is about _substitutability_. That is, if we need to supply a certain type, what other types can we substitute instead? Reference capabilities factor into this.
+## Simple subtypes
 
-## Simple substitution
+Subtyping is about _substitutability_. That is, if we need to supply a certain type, what other types can we substitute instead? Reference capabilities are one important component. We can start by going through a few simpler cases, and then we'll talk about the full chart.
 
-First, let's cover substitution without worrying about ephemeral types (`^`) or alias types (`!`). The `<:` symbol means "is a subtype of" or alternatively "can be substituted for".
+First, let's look at the four capabilities, `ref`, `val`, `box`, `tag`. These have a very useful property: they alias to themselves (and unalias to themselves, as well). This will make the subtyping a lot simpler to work with. Then we can talk about `iso` and `trn`.
```

Comment:
Small suggestion to the first sentence to make it clearer that these four aren't the entire set of capabilities available.

```suggestion
First, let's look at the four capabilities `ref`, `val`, `box`, and `tag`. These have a very useful property: they alias to themselves (and unalias to themselves, as well). This will make the subtyping a lot simpler to work with. Then we can talk about `iso` and `trn`.
```

Reply:
Good point, applied.